### PR TITLE
Static get form

### PR DESCRIPTION
**Pull request by thinklikeadesigner**
### [Trello card Static response for GET /api/form](https://trello.com/c/fJh8kr7b)

I added an object in frontend/src/utils/api.js that should mimic a get request for form using the following endpoint

```/api/form```

### CODE REVIEW

**Comment by thinklikeadesigner** [comment]
I corrected my misunderstanding. static response is sent from backend now

Screen Shot 2021-10-30 at 4 33 20 PM

File: backend/app.js [hunk 1]
```diff
@@ -16,15 +20,9 @@ app.use(cors());
 app.use(helmet());
 app.use(express.json());
 app.use(errors());
-
+app.use(rootRouter);
 mongoose.connect(
-  NODE_ENV === 'production' ? MONGODB_URI : 'mongodb://localhost:27017/newsexplorer',
```
Comment (vgalilei) [inline backend/app.js:21]: I think we have separate PR for this

File: backend/routes/forms.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+
+const express = require('express');
+const router = express.Router();
+
+router.get('/api/form', (req, res) => {
```
Comment (vgalilei) [inline backend/routes/forms.js:6]: Awesome!!!

File: frontend/src/components/footer/styledFooter.js [hunk 3]
```diff
@@ -1,57 +1,2 @@
-import styled from 'styled-components';
```
Comment (vgalilei) [inline frontend/src/components/footer/styledFooter.js:1]: I think we should solve only one issue per commit. So this commit is about the mock response to the form. So let it be here only this one.

File: backend/app.js (hunk 1 above)
Comment (thinklikeadesigner) [reply backend/app.js:21]: this is true 👍

File: frontend/src/components/footer/styledFooter.js (hunk 3 above)
Comment (thinklikeadesigner) [reply frontend/src/components/footer/styledFooter.js:1]: I think i accidentally pulled from main. will try to avoid that next time!

Comment (thinklikeadesigner) [reply frontend/src/components/footer/styledFooter.js]: kay, I unpolluted the PR. these changes are only related to server.

**Comment by thinklikeadesigner** [comment]
Okay I cleaned up this PR to only include server changes @vgalilei

File: backend/routes/forms.js (hunk 2 above)
Comment (thinklikeadesigner) [reply backend/routes/forms.js:6]: Thanks :)

**Review by vgalilei — APPROVED** [review]
LGTM

**Pull request merged**